Translation day01

massimilianoj opened the pull request:
Please let me know what you think. Some parts need to be reviewed. I
could not translate two sentences (I put in bold). Moreover I propose a
different style for link that I like more.

massimilianoj committed:
Translation day01

Please let me know what you think. Some parts need to be reviewed. I
could not translate two sentences (I put in bold). Moreover I propose a
different style for link that I like more.

kenjisato commented on 00-preface.Rmd:
```diff
@@ -11,7 +11,7 @@
 
 第2の課題は, 読者にコンピュータシミュレーションの基礎的な手法を身につけてもらうことである. 線形合理的期待モデルは近年のマクロ経済の数量分析に大きな役割を果たしているから, 各自の研究プロジェクトに大いに役に立つものと信じている. 
-数学もプログラミングも自分の手を動かし, 頭を悩ませなければ習熟できない. 理論とその実装を行き来しながら総合的な問題解決能力を養ってほしい. この講義では, 主に R言語を用いる.^[R ([https://www.r-project.org/](https://www.r-project.org/)) と RStudio ([https://www.rstudio.com/](https://www.rstudio.com/)) の最新版を各自インストールしてほしい. R言語を選んだ理由は, 計量経済学の学習のために利用したことのある学生が多いだろう考えたからであり, R がシミュレーションのために最適であるとは考えていない. 大変人気のあるプログラミング言語・環境であるから一度は学んでおいて損はないと思う.]
+数学もプログラミングも自分の手を動かし, 頭を悩ませなければ習熟できない. 理論とその実装を行き来しながら総合的な問題解決能力を養ってほしい. この講義では, 主に R言語を用いる.^[[R](https://www.r-project.org/) と [RStudio](https://www.rstudio.com/) の最新版を各自インストールしてほしい. R言語を選んだ理由は, 計量経済学の学習のために利用したことのある学生が多いだろう考えたからであり, R がシミュレーションのために最適であるとは考えていない. 大変人気のあるプログラミング言語・環境であるから一度は学んでおいて損はないと思う.]
```
I understand what you mean. I'm trying to make this work look nice in print. That's why I wanted to show the URLs.

kenjisato reviewed (commented):
Thanks for your contribution. I don't think the book down package allows to make internationalization happen. Can you please make a pull request against https://github.com/kenjisato/led-en ? Setting your remotes as follows would be fine.

origin = kenjisato/led-en
upstream = kenjisato/led

You don't have to rename the Rmd files.

massimilianoj committed:
Translation

I change the former day01 in day00 and I added a new day01. This to have
the same number in the original files.
